src/app/cli/src/serialization.ml

```diff
@@ -0,0 +1,21 @@
+open Core
```

Comment:
I don't think this module is used anywhere. ctrl-f `serialization` doesn't return any relevant results.